tweet_stream.py

```diff
@@ -21,11 +22,15 @@
 this.num_tweets = 100
 this.tweets = []
 this.trends = ["#"]
-
+this.st_time = time.time()
 
 class listener(StreamListener):
     def on_data(self, data):
 
+        if time.time() - this.st_time > 60:
+            log.error("[Tweet stream] on data timing out. Couldn't find tweets within 60 seconds")
```

Comment:
Did you mean to call on_timeout function here?

Reply:
no, on data is called whenever tweetstream receives the data. The logic here is by the time we start listening if we didn't receive 100 tweets or 60 seconds have elapsed return False and tweetstream will close.